test/e2e/onLCP-test.js

```diff
@@ -921,3 +904,12 @@ const assertFullReportsAreCorrect = (beacons) => {
   assert(lcp2.entries[0].startTime > lcp1.entries[0].startTime);
   assert.match(lcp2.navigationType, /navigate|reload/);
 };
+
+const switchToNewTabAndBack = async () => {
```

Comment:
👍 we probably should have been doing something like this the whole time.

Reply:
Yeah I only recently added a test to do an actual tab change in #612 (and you also liked it then!) so before that we didn’t have anything like this.